core/src/main/resources/messages/messages.properties

```diff
@@ -0,0 +1,472 @@
+test.message=Hi my dear friend
```

Comment:
The message.properties are added for core package to access the messages for `CloudbreakMessagesHostServiceTypeTest`

Reply:
Sorry, I really do not get this one. The same messages are already available in [the similarly named](https://github.com/hortonworks/cloudbreak/blob/master/common/src/main/resources/messages/messages.properties) file of the `common` module.